internal/testcontext/compile.go

```diff
@@ -11,6 +11,8 @@ import (
 	"testing"
 )
 
+var CLibMath = Include{Library: "m", Installed: true}
```

Comment:
Why do we need math?

Reply:
Just in the tests, same goes for `time.h`. We're using math to exponentially increase the size of the test upload; I suppose there are other ways to do it. I don't have a strong opinion on it.

Reply:
Yeah, `i * i`.

Reply:
can/should we keep the standard library include bit?